code/templates/annex-a.html

```diff
@@ -1,15 +1,37 @@
 {% extends "main.html" %}
 {% block pagecontent %}
 
-<h1>Annex A Computer interpretable listings</h1>
+<h1 style="text-align:center">
+  <span>Annex A</span><br>
+  <span style="font-weight:lighter">(normative)</span><br><br>
+  <span>Computer interpretable listings</span><br><br></h1>
 
 <p>
-    This annex contains a listing of the complete schema combining all definitions of clauses 5, 6, 7, and 8 without comments or other explanatory text. These listings are available in computer-interpretable form that may be parsed by computer.
+    This annex contains a listing of the complete schema combining all definitions of clauses 5, 6, 7, and 8 without comments or other explanatory text.
+    These listings are available in computer-interpretable form that may be parsed by computer. The schemas are governed by the following schema definition languages that shall be adhered to when implementing.
 </p>
+
+<ul>
+  <li>EXPRESS schema following ISO 10303-11: <i>Industrial automation systems and integration — Product data representation and exchange — Part 11: description methods: The EXPRESS Language Reference Manual</i>
+    and its serialisation following ISO 10303-21: <i>Industrial automation systems and integration — Product data representation and exchange — Part 21: Implementation methods: Clear text encoding of the exchange structure</i>
```

Comment:
Strictly speaking, part 21 is not relevant right for Annex A?

Reply:
Yes but we are requested to name it as a requirement in the whole document (refer to it with 'shall'). So I did it here.

Reply:
One more comment. Should ISO 10303-28 also not be listed then?